mirrord/agent/src/file.rs

```diff
@@ -2,13 +2,16 @@ use std::{
     self,
     collections::HashMap,
     fs::{File, OpenOptions, ReadDir},
+    io,
     io::{prelude::*, BufReader, SeekFrom},
     iter::Enumerate,
-    os::unix::prelude::{DirEntryExt, FileExt, MetadataExt},
+    os::unix::prelude::FileExt,
     path::{Path, PathBuf},
 };
 
 use faccess::{AccessMode, PathExt};
+#[cfg(target_os = "linux")]
```

Comment:
currently agent doesn't compile for macOS, so not sure how helpful is adding those stuff.